src/wagmi/index.ts

```diff
@@ -1,2 +1,2 @@
 export * from './connector'
-export * from './rainbowkit/connector'
```

Comment:
I think it is a bit weird that there is not a mention of rainbowkit anymore in the folder, the wallet implementation is specific or rainbowkit ? Or Am I mistaking ?